modules/nodes-pods-vertical-autoscaler-using-about.adoc

```diff
@@ -91,6 +91,39 @@ The output shows the recommended resources, `target`, the minimum recommended re
 
 The VPA uses the `lowerBound` and `upperBound` values to determine if a pod needs to be updated. If a pod has resource requests below the `lowerBound` values or above the `upperBound` values, the VPA terminates and recreates the pod with the `target` values.
 
+[id="nodes-pods-vertical-autoscaler-using-one-pod_{context}"]
+== About changing the VPA minimum value
+
+By default, workload objects must specify a minimum of two replicas in order for the VPA to monitor and update pods. As a result, workload objects that specify fewer than two replicas are not acted upon by the VPA. You can change this minimum value by modifying the `VerticalPodAutoscalerController` object to add the `minReplicas` parameter.
```

Comment:
I suggest that you remove `monitor and`. The VPA recommender will monitor any configured workload, regardless of the number of replicas, so we don't want to give the impression that it won't.

I also suggest changing `update pods.` to `automatically update pods.`